tests/Keboola/DebugLogUploader/UploaderFileTest.php

```diff
@@ -64,24 +60,25 @@ public function testUploadStringJson()
 "codeclimate/php-test-reporter":"~0.3"}\n\n}\n\n
 JSON;
         $contentJsonFormatted = <<<JSON
+
 {
-    "require": {
-        "php": "~5.6",
-        "aws/aws-sdk-php": "~3.18",
-        "symfony/filesystem": "~3.0"
-    },
-    "autoload": {
-        "psr-4": {
-            "Keboola\\\\": "src/Keboola/"
-        }
-    },
-    "require-dev": {
-        "symfony/process": "~3.0",
-        "phpunit/phpunit": "~5.3",
-        "squizlabs/php_codesniffer": "~2.6",
-        "codeclimate/php-test-reporter": "~0.3"
-    }
+	"require":{
+		"php":"~5.6",
+		"aws/aws-sdk-php":"~3.18",
+		
+"symfony/filesystem":"~3.0"
+
+	},
+	"autoload":{
+		"psr-4":{
+			"Keboola\\\":"src/Keboola/"}},"require-dev":{
+
+"symfony/process":"~3.0","phpunit/phpunit":"~5.3","squizlabs/php_codesniffer":"~2.6",
+"codeclimate/php-test-reporter":"~0.3"}
+
 }
+
+
 JSON;
```

Comment:
Tady to vůbec nechápu je ten formát dost takový dodrbaný. ani vlastně pořádně nechápu ten usecase jak sa používá ten Json::prettyPrint nevíš @tomasfejfar ?

Reply:
smazal sem to